agent/06_agent_create_cluster.sh

```diff
@@ -95,6 +95,19 @@ function create_agent_iso_no_registry() {
   popd
 }
 
+function assert_agent_no_registry_iso_size(){
+  agent_iso_no_registry=$(get_agent_iso_no_registry)
+  iso_size=$(stat -c%s "$agent_iso_no_registry")
+  
+  # With 4.19 tech preview, the expected ISO size is approximately 36GB
+  iso_size_limit=$(($AGENT_OVE_ISO_SIZE * 1024 * 1024 * 1024))
+
+  if (( iso_size > iso_size_limit )); then
+    echo "Error: OVE ISO size of $agent_iso_no_registry exceeds ${AGENT_OVE_ISO_SIZE}GB"
```

Comment:
Let's have the message also print out the iso_size so that we can see by how much it is over.